node/pkg/alephium/watcher.go

```diff
@@ -385,16 +388,17 @@ func (w *Watcher) handleEvents_(
 			}
 
 			remain := make([]*UnconfirmedEvent, 0)
-			logger.Info("processing events from block", zap.String("blockHash", blockEvents.header.Hash), zap.Int("size", len(blockEvents.events)))
+			logger.Info(
+				"processing events from block",
+				zap.String("blockHash", blockEvents.header.Hash),
+				zap.Int32("blockHeight", blockEvents.header.Height),
+				zap.Int64("blockTimestamp", blockEvents.header.Timestamp),
+				zap.Int("size", len(blockEvents.events)),
+			)
 			for _, event := range blockEvents.events {
-				eventConfirmations := event.msg.consistencyLevel
-				if blockEvents.header.Height+int32(eventConfirmations) > height {
-					logger.Debug(
-						"event not confirmed",
-						zap.String("txId", event.TxId),
-						zap.Int32("blockHeight", blockEvents.header.Height),
-						zap.Uint8("confirmations", eventConfirmations),
-					)
+				consistencyLevel := event.msg.consistencyLevel
+				if !isEventConfirmed(consistencyLevel, blockEvents.header, now, height) {
+					logger.Debug("event not confirmed", zap.String("txId", event.TxId), zap.Uint8("consistencyLevel", consistencyLevel))
```

Comment:
[Nit] Would it easier to read the log if we log inside the `isEventConfirmed`, so we know if it's not confirmed due to height or timestamp?

Reply:
Sounds great 👍, will update.